### PR TITLE
Change should help with running make installcheck

### PR DESCRIPTION
**Pull request by calewis**
`make installcheck` was failing on our CI because `libmacro.so` couldn't be discovered.

### CODE REVIEW

File: tests/api/mpi/testexec.in [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 export PYTHONPATH=$PYTHONPATH:@prefix@/include/python
 export SST_LIB_PATH=@prefix@/lib:$SST_LIB_PATH
+export LD_LIBRARY_PATH=@prefix@/lib:$LD_LIBRARY_PATH
```
Comment (jpkenny) [inline tests/api/mpi/testexec.in:5]: Should be like sstmac-check.in:

export PYTHONPATH=@abs_top_srcdir@/python:$PYTHONPATH
export SST_LIB_PATH=@abs_top_builddir@/sstmac/install/.libs
export DYLD_LIBRARY_PATH=@abs_top_builddir@/sstmac/install/.libs:$DYLD_LIBRARY_PATH
export DYLD_LIBRARY_PATH=@abs_top_builddir@/sst-dumpi/dumpi/libundumpi/.libs:$DYLD_LIBRARY_PATH
export LD_LIBRARY_PATH=@abs_top_builddir@/sstmac/install/.libs:$LD_LIBRARY_PATH
export LD_LIBRARY_PATH=@abs_top_builddir@/sst-dumpi/dumpi/libundumpi/.libs:$LD_LIBRARY_PATH

**Review by jpkenny — COMMENTED** [review]
And shouldn't we fix registration while we're at it?

File: tests/api/mpi/testexec.in (hunk 1 above)
Comment (calewis) [reply tests/api/mpi/testexec.in:5]: Okay, except with `prefix` instead of build?

Comment (jpkenny) [reply tests/api/mpi/testexec.in:6]: Yes, for testexec it should be wherever things get installed.
Don't forget the sst-dumpi path, libmacro depends on libundmpi

Comment (jpkenny) [reply tests/api/mpi/testexec.in:6]: Ok, dumpi and macro prefix are the same. Resolved.

**Review by jpkenny — APPROVED** [review]
LGTM

**Comment by calewis** [comment]
@jpkenny I think the pre-merge inspection is messed up because I started with master and switched to devel

**Pull request merged**